Library/Homebrew/cmd/audit.rb

```diff
@@ -855,6 +855,14 @@ def audit_line(line, lineno)
       if line =~ /(require ["']formula["'])/
         problem "`#{$1}` is now unnecessary"
       end
+
+      if line =~ /#\{share\}\/#{formula.name}/
```

Comment:
Why not use a cheaper `String#include?` check, i.e. `line.include?("\#{share}/#{formula.name}")`, if you don't really have a regular expression here and are not capturing any of the sub-expressions?